enterprise/app/trends/percentile_chart.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+
+import * as format from "../../../app/format/format";
+import { ResponsiveContainer, ComposedChart, CartesianGrid, XAxis, YAxis, Line, Legend, Tooltip } from "recharts";
+
+interface Props {
```

Comment:
nit: we aren't 100% consistent on this yet, but since the component is exported, the props type should be exported too as the props are part of the public API (and prefixed w/ component name to avoid name collisions)

```suggestion
export interface PercentilesChartProps {
```